playground/mocks/data/idp/idx/enroll-profile-new-additional-fields.json

```diff
@@ -61,6 +62,7 @@
                       "value":{
                         "type":"object",
                         "value":{
+                          "format": "country-code",
                           "inputType":"text"
```

Comment:
Just curious, why are we allowing text as an input for country-code?

Reply:
Good question. because we need to store admin's settings. every time after admin choose `text` display, next time, when admin comes back to the settings, the `text` option can be remembered. Previously, we do not add it as it is not needed in SIW and just an default value in ui. but for country code, it is one of the radio option in ui.

Reply:
Oh okay, so it's another option we support then. Got it!